django/library/fs.py

```diff
@@ -448,8 +452,12 @@ def initialize_codemeta(self, path=None, metadata=None):
             json.dump(metadata, codemeta_out)
         return True
 
+    def retrieve_review_archive(self):
+        shutil.make_archive(str(self.rootdir.joinpath('review_archive')), format='zip', root_dir=str(self.sip_contents_dir))
```

Comment:
Not a huge fan of having multiple ways to generate the review_archivepath components with and without the .zip extension but that's the price of `shutil.make_archive` I guess..